From 650af2d9bb7765e39105724692aac9b93108761d Mon Sep 17 00:00:00 2001
From: Tintu Thomas <tintu.thomas@arm.com>
Date: Tue, 21 Mar 2023 15:09:08 +0000
Subject: [PATCH] platform: tc: change the host uart to the TC RoS SoC UART

* Changing the host UART from AP NS UART since AP NS logs are now
  using it.
* With this, RSS uses RoS SoC UART at 0x7FF80000UL

Signed-off-by: Tintu Thomas <tintu.thomas@arm.com>
---
 platform/ext/target/arm/rss/tc/host_base_address.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform/ext/target/arm/rss/tc/host_base_address.h b/platform/ext/target/arm/rss/tc/host_base_address.h
index 2acb97e45..356b51169 100644
--- a/platform/ext/target/arm/rss/tc/host_base_address.h
+++ b/platform/ext/target/arm/rss/tc/host_base_address.h
@@ -29,7 +29,7 @@
 #define SCP_BOOT_SRAM_BASE  0x40000000UL /* SCP initial boot SRAM base address */
 #define SCP_BOOT_SRAM_SIZE  0x80000U     /* 512KB */
 
-#define HOST_UART_BASE     0x2A400000UL /* Host UART base address */
+#define HOST_UART_BASE     0x7FF80000UL /* Host UART base address */
 #define HOST_UART_SIZE     0x2000U      /* 8KB */
 
 #define HOST_FLASH0_BASE   0x08000000UL /* Host flash base address */
-- 
2.39.1

